packages/terra-collapsible-menu-view/src/terra-dev-site/doc/collapsible-menu-view/UpgradeGuide.3.doc.mdx

```diff
@@ -0,0 +1,16 @@
+import { Badge } from 'terra-collapsible-menu-view/package.json?dev-site-package';
```

Comment:
In most of these PRS, the order was 
- readme
- upgrade guide
- changelog

can we try to maintain this patterns so docs are organized consistently?

Reply:
Changed Order 946c29c